### DISCUSSION TITLE
Seplos BMS active balance board: How to get cell info?

### DISCUSSION THREAD

**Discussion by broekieman** [mr-manuel/venus-os_dbus-serialbattery#103] [General]
How to get cell info from Seplos Bms via the active balance board and parallel connections multiple packs?

Seplos Bms has cell info per group of 4 cells. The active balance board has a sensor on each cell. Would it be possible to read these values?

Seplos does not read 2 bms when parallel connected. In DebBMSSudio i can not read 2 packs parallel connected. Does anyone have a clue to get this working?

**Comment by mycroft2k** [comment]
If it's a Seplos BMS V3, switch the mode from 'Standalone Mode' to 'Parallel Mode' in the bottom right corner of BMS-STUDIO V2.0.0.1

![Parralel Modus](https://github.com/user-attachments/assets/5b016a7a-7123-4567-a4d1-2918d1fe6cb7)

**For the active balancer board, you need a TTL adapter.**

The connector is located on the far right, and the pins are assigned as follows:
```
PIN 1: GND
PIN 2: TXD
PIN 3: RXD
PIN 4: 3.3V (not needed)
```
![image (1)](https://github.com/user-attachments/assets/16f7d592-7705-4ea8-9deb-b390082b40c3)


**After connecting, the BMS Studio can be started with the ABU.**

```
Baud rate: 19200
Password: 88888888
```
![image](https://github.com/user-attachments/assets/648f0a1c-8ac3-404d-bd60-614c9a0273ad)

**Notes:**

```
All parameters must be loaded manually using the blue arrow.
The "Read All" function does not work because the last parameter is missing.
```
![image](https://github.com/user-attachments/assets/6af05570-e706-434a-8f0f-866c314283da)
